__tests__/app.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require('supertest')
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+const db = require('../db/connection');
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    if (db.end) db.end();
+});
+
+describe("/api test suite", () => {
+    test("200: should respond with an okay message", () => {
+    return request(app)
+    .get('/api')
+    .expect(200)  
+    .then(({body}) => {
+        expect(body.msg).toBe("Everything is working fine!")
+        })
+    })
+});
```

Comment:
The ticket hasn't asked for this behaviour above, you will have to change this for your next task as we have functionality in mind for the /api endpoint!